lib/sdf-server/src/service/diagram/create_component.rs

```diff
@@ -82,12 +82,6 @@ pub async fn create_component(
 
             let variant_id = Schema::get_or_install_default_variant(&ctx, schema_id).await?;
             let variant = SchemaVariant::get_by_id(&ctx, variant_id).await?;
-            let managed_schema_ids = SchemaVariant::all_managed_schemas(&ctx, variant_id).await?;
```

Comment:
This is where we would auto-install managed schemas when dragging an asset onto the canvas. Now this happens when the management function runs and creates a component whose asset is not yet installed.